app.go

```diff
@@ -151,6 +151,17 @@ func Logger(p Printer) Option {
 	})
 }
 
+// NopLogger disables the application's log output.
+func NopLogger() Option {
+	return Logger(nopLogger{})
```

Comment:
I don't feel strongly about this but: I'm not sure this needs to be a function. It can just be a constant option.

```
app := fx.New(fx.NopLogger, ....)
```